BrightID/src/reducer/socialMediaSlice.ts

```diff
@@ -49,6 +52,7 @@ const socialMediaSlice = createSlice({
           id: incomingId,
           changes: {
             profile: incomingProfile,
+            brightIdSocialAppData: incomingBrightIdSocialAppData,
```

Comment:
This allows to override token and contextId of existing entries. It may be better to prevent this, otherwise user will not be able to update or delete his data in the backend again? I'm not sure if this is necessary though.

Reply:
I'm using this to update the token and syncing status.